Rakefile

```diff
@@ -5,12 +5,3 @@
 require_relative 'config/application'
 
 Rails.application.load_tasks
-
-if Rails.env.development?
```

Comment:
This was causing some loading issues. Since we don't use this/it's currently broken I've gone and removed it